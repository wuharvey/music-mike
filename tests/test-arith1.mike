Printint(4);
Printint(4+4);
Printint(2*8);
Printint(10/2);

