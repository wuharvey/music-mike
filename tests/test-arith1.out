4
8
16
5
